lib/foreman_bootdisk/engine.rb

```diff
@@ -31,11 +31,11 @@ class Engine < ::Rails::Engine
         requires_foreman '>= 1.9'
 
         security_block :bootdisk do |map|
-          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :help],
+          permission :download_bootdisk, {:'foreman_bootdisk/disks' => [:generic, :host, :full_host, :subnet, :help],
                                           :'foreman_bootdisk/api/v2/disks' => [:generic, :host]}
         end
 
-        role "Boot disk access", [:download_bootdisk]
+        role "Boot disk access", [:download_bootdisk, :view_hosts, :view_subnets]
```

Comment:
I don't know why this fails:
    /var/lib/workspace/workspace/test_plugin_pull_request@2/database/postgresql/ruby/1.9.3/foreman/app/models/role.rb:131:in `add_permissions': some permissions were not found (ArgumentError)

The new permissions are added via core seeds... strange.